dev/docker-compose-integration.yml

```diff
@@ -26,15 +26,13 @@ services:
       - rest
       - hive
       - minio
-    volumes:
-      - ./warehouse:/home/iceberg/warehouse
     environment:
       - AWS_ACCESS_KEY_ID=admin
       - AWS_SECRET_ACCESS_KEY=password
       - AWS_REGION=us-east-1
     ports:
-      - 8888:8888
-      - 8080:8080
```

Comment:
removed port 8888 that was previously used for notebooks
replaced spark master web ui (8080) with spark app ui (4040)